precompiles/bank/Bank.sol

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+address constant BANK_PRECOMPILE_ADDRESS = 0x0000000000000000000000000000000000001001;
+
+IBank constant BANK_CONTRACT = IBank(
+    BANK_PRECOMPILE_ADDRESS
+);
+
+interface IBank {
+    // Transactions
+    function send(
+        address fromAddress,
+        address toAddress,
+        string memory denom,
```

Comment:
for my education: what are some examples of what denom could be?

Reply:
It'd be one of `usei`, `ibc/<route1>/<route2>/....` tokens, `tokenfactory/xyz...` tokens